biscuit-auth/src/token/third_party.rs

```diff
@@ -124,7 +126,7 @@ impl ThirdPartyRequest {
         let mut symbols = SymbolTable::new();
         symbols.public_keys = self.public_keys.clone();
         let mut block = block_builder.build(symbols);
-        block.version = super::MAX_SCHEMA_VERSION;
+        block.version = max(4, block.version);
```

Comment:
we should probably have a list of constants somewhere like `SCHEMA_VERSION_3RD_PARTY = 4` to track those checks more easily

Reply:
I can take care of that after merging that one

Reply:
I had the exact same thought, yep

Reply:
added in https://github.com/biscuit-auth/biscuit-rust/pull/217/commits/c62aeabda3b0fe98d12630fbe40b327b2b7c0917